### PR TITLE
mailcap: 2.1.53 -> 2.1.54

### PR DESCRIPTION
**Pull request by raphaelr**
###### Description of changes

Changelog: https://pagure.io/mailcap/blob/master/f/NEWS
Diff: https://pagure.io/mailcap/c/9699055a1b4dfb90f7594ee2e8dda705fa56d3b8?branch=master

Notably, this updates the mime type of `.js` files from `application/javascript` to `text/javascript`.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by FliegendeWurst — APPROVED** [review]

File: nixos/modules/services/video/frigate.nix [hunk 1]
```diff
@@ -271,7 +271,7 @@ in
               sub_filter '"/BASE_PATH/monacoeditorwork/' '"$http_x_ingress_path/assets/';
               sub_filter 'return"/BASE_PATH/"' 'return window.baseUrl';
               sub_filter '<body>' '<body><script>window.baseUrl="$http_x_ingress_path/";</script>';
-              sub_filter_types text/css application/javascript;
+              sub_filter_types text/css text/javascript;
```
Comment (FliegendeWurst) [inline nixos/modules/services/video/frigate.nix:274]: Hold on, why was this mime type changed? It doesn't look related to mailcap?

Comment (raphaelr) [reply nixos/modules/services/video/frigate.nix:274]: mailcap, which is a dependency of nginx, changed the MIME type of .js files from `application/javascript` to `text/javascript` (see the diff linked in the PR description). So I changed the usages of the old mime type to match.

However, I looked at this half a year ago. There may have been more usages added since then. I'd have to revisit this update, and I don't have time currently. Closing this for now.

Comment (FliegendeWurst) [reply nixos/modules/services/video/frigate.nix:274]: > mailcap, **which is a dependency of nginx**

I didn't know that :innocent:. The changes make more sense then.

> However, I looked at this half a year ago. There may have been more usages added since then. I'd have to revisit this update, and I don't have time currently. Closing this for now.

Understandable.
